games-util/springlobby/springlobby-0.269.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+WX_GTK_VER="3.0"
+inherit cmake-utils gnome2-utils wxwidgets
+
+DESCRIPTION="The official lobby client for SpringRTS community games"
+HOMEPAGE="https://springlobby.springrts.com"
+SRC_URI="https://springlobby.springrts.com/dl/stable/${P}.tar.bz2"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug +libnotify +nls +sound"
+
+RDEPEND="
+	>=dev-libs/boost-1.35:=
```

Comment:
The version requirement can be dropped for a cleaner ebuild, it's always satisfied in ::gentoo tree.